### ISSUE TITLE
Use different name for model type

### ISSUE THREAD

**Issue by davidanthoff** [julia-mpsge/MPSGE.jl#130]
Don't overwrite Model. Make the primary data structure be called MPSGE. Using this I will have several models running around, one to calibrate the data, one algebraic version of the model, and one MPSGE. I don't want to have to specify `JuMP.Model` and `MPSGE.Model`. 

I would recommend a function `build_model(M::MPSGE)` that returns a JuMP model. MPSGE could remember this model for later use.

_Originally posted by @mitchphillipson in https://github.com/anthofflab/MPSGE.jl/issues/126#issuecomment-1708900880_

**Comment by davidanthoff** [comment]
Yeah, good point, that one might want to use `using JuMP, MPSGE`, and then things are not easy... `MPSGE` won't work, one can't use the same name for the module and the type, AFAIK. But maybe `MPSGEModel` might be a good choice? The only time one really needs to type that is during the constructor call, so not a big problem if it is not super short.

> I would recommend a function `build_model(M::MPSGE)` that returns a JuMP model.

**Comment by mitchphillipson** [comment]
This is implemented as `MPSGEModel` at the moment. Can discuss and change, if needed, sooner rather than later.

**Comment by mitchphillipson** [comment]
Closing issue. `MPSGEModel` is the default.
